### DISCUSSION TITLE
Twemoji on packages

### DISCUSSION THREAD

**Discussion by igorwastaken** [skyra-project/discord-components#123] [Ideas] [answered]
# Idea Twemoji on packages
I was thinking of something really cool to create, and I thought of Twemoji right here.

How this could look:
![example](https://user-images.githubusercontent.com/80823011/132128129-b0fccc9b-0371-4af5-9206-4fec8662d87f.jpg)

Let's literally create a thread about this here :)

**Comment by favna** [comment, marked as answer]
Imagine using Mee6 and not Skyra, FeelsBadMan /s 

Twemojis can be referenced by SVG such as https://raw.githubusercontent.com/twitter/twemoji/master/assets/svg/1f522.svg for 🔢. Because it's an SVG you can then pass your own width and height and you should be able to fit it along the text I think.

In HTML:
```html
<img src="https://raw.githubusercontent.com/twitter/twemoji/master/assets/svg/1f522.svg" width="800" height="200">
```


Edit: As for how to get the codepoint, there are 2 ways.

**Through JS**

If you can copy the emoji (i.e. from Emojipedia) you can execute this:
```js
[..."🔢"].map(c => c.codePointAt(0).toString(16));
```
Replace `🔢` with your emoji.

**Through the web**
Follow this:

https://user-images.githubusercontent.com/4019718/132128430-8fc1a36e-d344-47fd-9bb4-9be69bf38a66.mp4

**Reply by igorwastaken** [reply]
It can support any file type like `svg`, `png`, etc. I think...

**Reply by igorwastaken** [reply]
And Mee6 was just an example, and it was the first bot I thought of 😅

**Reply by favna** [reply]
> It can support any file type like `svg`, `png`, etc. I think...

Yes the PNG URL is: https://raw.githubusercontent.com/twitter/twemoji/master/assets/72x72/2b07.png
and the SVG URL is: https://raw.githubusercontent.com/twitter/twemoji/master/assets/svg/2b07.svg

The GitHub repo is: https://github.com/twitter/twemoji

My point is that because discord-components is used on the web this isn't a feature that needs to be added to the lib itself. You can just reference the images on the twemoji GitHub repo.

**Reply by igorwastaken** [reply]
ooh ok thanks for answering
